mlir/tools/rocmlir-gen/rocmlir-gen.cpp

```diff
@@ -1746,6 +1938,16 @@ createCPUConvWithCPP(ModuleOp module, func::FuncOp &func,
                  paddingWidthRightConstantOp, dilationHeightConstantOp,
                  dilationWidthConstantOp, accelConstantOp});
 
+  if (!filter.dyn_cast<BlockArgument>())
```

Comment:
And undo at the end.

Reply:
These `dyn_cast`s can be `isa<>`s instead